mlflow/pyfunc/model.py

```diff
@@ -461,7 +832,28 @@ def predict_stream(
         """
         Evaluates a ChatAgent input and produces a ChatAgent output. Wrapped to allow you to pass
```

Comment:
The "wrapped to allow" part is a little confusing/seems like an implementation detail, maybe we can mention the fact that dict input is also supported further down with an example, like what we did in the mlflow.pyfunc docs with

> In addition to calling predict and predict_stream methods with an input matching their type hints, you can also use a single dict that is of the [ChatAgentRequest](https://output.circle-artifacts.com/output/job/0e4ec2a5-1755-463e-8c46-3e9434a7e548/artifacts/0/docs/build/api_reference/python_api/mlflow.types.html#mlflow.types.agent.ChatAgentRequest) schema for ease of testing.